frontend/pages/medical-recovery-centre/index.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+
+const MedicalRecoveryCentre = () => {
+  return (
+    <>
+      {style}
+      <div className="container">
+        <h1>Medical Recovery Centre</h1>
+        <TheCottage />
+        <TierneyHouse />
+        <TBAPerth />
```

Comment:
This is some interesting component division. I should really do something like this, as it makes it very easy to read the sections.

Reply:
Glad you were able to learn something me for once :D